src/site/content/en/blog/mediastreamtrack-insertable-media-processing/index.md

```diff
@@ -0,0 +1,252 @@
+---
+layout: post
+title: Insertable streams for MediaStreamTrack
+subhead: |
+  The contents of MediaStreamTrack objects exposed as a collection of streams so they can be manipulated
+  to introduce new components.
+authors:
+  - thomassteiner
+date: 2021-04-26
+description: |
+  Insertable streams for MediaStreamTrack expose the contents of a MediaStreamTrack as a collection
+  of streams (as defined by the WHATWG Streams API), which can be manipulated to introduce new
+  components.
+hero: image/8WbTDNrhLsU0El80frMBGE4eMCD3/Qu2wfQ3pxR8AeEfty88S.jpg
+alt: Cup of coffee and a laptop with a video conference running showing many participants.
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - media
+  - capabilities
+---
+
+{% Aside %} Insertable streams for `MediaStreamTrack` is part of the
+[capabilities project](https://web.dev/fugu-status/) and is currently in development. This post will
+be updated as the implementation progresses. {% endAside %}
+
+## Background
+
+The
+[Media Capture and Streams API](https://developer.mozilla.org/en-US/docs/Web/API/Media_Streams_API),
+often called the Media Streams API or MediaStream API, is an API related to
+[WebRTC](https://developer.mozilla.org/en-US/docs/Web/API/WebRTC_API) which provides support for
+streaming audio and video data.
+
+The API is based on the manipulation of a
```

Comment:
Can you flip this section around to focus on the MediaStreamTrack, and just mention briefly at the end that a MediaStream is a collection of audio and video tracks, used to connect MediaStreamTracks to an HTML video element?

Many argue that the MediaStream was a design mistake, and we should focus our APIs on MediaStreamTracks; the MST transform API was heavily influenced by this, so the focus should be on MediaStreamTracks in this article.

Reply:
Addressed in https://github.com/GoogleChrome/web.dev/pull/5092/commits/d014d7d54703c7fe0b256dfcbc30756e373645db.

Reply:
You still start off with Mediastream and introduce MediaStreamTrack later.
I want you to start off with MediaStreamTrack and have a closing paragraph mentioning MediaStream.

Reply:
The idea was to explain top down the concepts. I’ll see what I can do tomorrow.

Reply:
@alvestrand, PTAL. As of https://github.com/GoogleChrome/web.dev/pull/5092/commits/a787026b2d799f3e6eccd050e789480677e4d694, I now lead with `MediaStreamTrack`.